src/explore-education-statistics-admin/src/pages/publication/components/PublicationDetailsForm.tsx

```diff
@@ -71,6 +72,15 @@ export default function PublicationDetailsForm({
     onSubmit();
   };
 
+  const cancelConfirmModal = () => {
+    toggleConfirmModal.off();
+    // This setTimeout is needed because the focus() fn needs to be called
+    // on the next tick. Without it, the button doesn't focus.
+    // I think the modal tries to handle the focus on close, which prevents
+    // our custom focus handling from working at that point.
+    setTimeout(() => submitButtonRef.current?.focus(), 0);
+  };
```

Comment:
Let's use a useEffect instead of setTimeout, to automatically set focus every time the modal closes
```
  useEffect(() => {
    if (showConfirmModal === false) {
      submitButtonRef.current?.focus();
    }
  }, [showConfirmModal]);
```